src/chain_spec.rs

```diff
@@ -210,6 +213,24 @@ fn polymath_mem(n: u8) -> GenesisIdentityRecord<AccountId> {
 
 const STASH: u128 = 5_000_000 * POLY;
 const ENDOWMENT: u128 = 100_000_000 * POLY;
+const BOOTSTRAP_STASH: u128 = 10_000 * POLY;
+const BOOTSTRAP_TREASURY: u128 = 30_000_000 * POLY;
+
+fn bridge_lock(nonce: u32, hash: &'static str) -> (u32, H256) {
+    let offset = if hash.starts_with("0x") { 2 } else { 0 };
+    let stripped_hash = &hash[offset..];
+    let hash_vec: Vec<u8> = rustc_hex::FromHex::from_hex(stripped_hash)
+        .expect("Failed to decode transaction hash (Invalid hex Value)");
+    let hash_array: [u8; 32] = hash_vec
+        .try_into()
+        .expect("Failed to decode transaction hash (Invalid hash length)");
+    (nonce, hash_array.into())
+}
+
+fn generate_bridge_locks(count: u32) -> Vec<(u32, H256)> {
```

Comment:
It makes sense to define a struct with named fields and some methods for the element type `(u32, H256)`.